### DISCUSSION TITLE
[Bug] Typescript does not use the same trigger characters as VS Code

### DISCUSSION THREAD

**Discussion by schiem** [microsoft/monaco-editor#3711] [Q&A]
### Reproducible in vscode.dev or in VS Code Desktop?

- [x] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Code

```typescript
monaco.editor.create(document.getElementById('container'), {
	value: "type Foo = 'foo' | 'bar';\nconst doSomething = (foo: Foo): void => {\n    // Do something\n}\ndoSomething('')",
	language: 'typescript'
});
```

### Reproduction Steps

1. Create a type in Monaco that consists of string literals
2. Create a function which uses that type as a parameter
3. Attempt to call that function

### Actual (Problematic) Behavior

Autocomplete does not appear until you manually bring it up with ctrl + space, both after typing the opening parenthesis and the opening quote.

### Expected Behavior

Autocomplete (with options 'foo' and 'bar', in the case of the example) appears as soon as you type a quote in the function parameter.  This is how it VS Code works, both in the desktop and the browser.

### Additional Context

This seems to be caused by the different triggers characters in the `SuggestAdapter`, [in Monaco](https://github.com/microsoft/monaco-editor/blob/a385674a34a33947483e0f7266b5a7dbcefe9b9d/src/typescript/languageFeatures.ts#L435-L438) and [in VS Code](https://github.com/microsoft/vscode/blob/e7b3724e0cdc9095d73a06b47203baa2c1777ea8/extensions/typescript-language-features/src/languageFeatures/completions.ts#L642).  It's possible that this is intentional behavior, but I couldn't find it documented anywhere.

This issue was brought up in #2883, but was not marked as a bug there.

**Comment by jacks-bush** [comment]
I would also like this to be added
